### ISSUE TITLE
HTTP Request Tool where enabling Optimise Response for HTML errors

### ISSUE THREAD

**Issue by jimleuk** [n8n-io/n8n#11385]
### Bug Description

When enabling "Optimise Response" option for HTML in the HTTP request tool, the user recieves the following error as tool output. This prevents the AI agent from using the tool correctly.
```
{
  "response": "There was an error: \"Cannot read properties of undefined (reading 'load')\""
}
```

Disabling the "Optimise Response" option allows the tool to function as expected.

### With "Optimise Response" option
<img src="https://github.com/user-attachments/assets/7e0b5a69-14e8-4ef7-ba41-6b713fe4c9ae" width="480" />

### Without "Optimise Response" option
<img src="https://github.com/user-attachments/assets/ef844cb9-07d1-42c2-b417-10cb999ebe4a" width="480" />

### HTTP Request Tool Options
<img src="https://github.com/user-attachments/assets/8ed0cd0b-bfed-4fb2-9bcf-f79f9d07c54e" width="320" />

### To Reproduce

1. Create Tools agent and attach HTTP Request Tool
2. Define URL placeholder for agent to input desired web address to visit
3. Enable "Optimise Response" option and select "HTML" from dropdown. Use default "body" as the content to extract.
4. Test agent
5. HTTP request tool fails to retrieve the contents of the URL.

### Expected behavior

With "Optimise Response" option enabled for HTML, I'd expect the HTTP request tool to return the body of HTML of the given URL.

### Operating System

MacOS Sonoma

### n8n Version

1.64.0

### Node.js Version

???

### Database

SQLite (default)

### Execution mode

main (default)

**Comment by Joffcom** [comment]
Hey @jimleuk,

We have created an internal ticket to look into this which we will be tracking as "GHC-342"

**Comment by janober** [comment]
Fix got released with [`n8n@1.66.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.66.0)
